src/Components/CategoryInput/CategoryInput.tsx

```diff
@@ -1,16 +1,15 @@
 import { useEffect, useRef, useState } from 'react'
```

Comment:
since we removed .scss files, could you take out all .tsx files outside of each folder and remove them?

Reply:
okay